packages/flutter/lib/src/material/input.dart

```diff
@@ -47,6 +47,7 @@ class InputField extends StatefulWidget {
     this.keyboardType: TextInputType.text,
     this.hintText,
     this.style,
+    this.hintStyle,
```

Comment:
Sounds good.

I was going to add them to the other Text widgets, but technically they are based on "Material Design Specs". Is it fine to provide this override?